components/public-api/gitpod/experimental/v2/workspace.proto

```diff
@@ -0,0 +1,260 @@
+syntax = "proto3";
+
+package gitpod.experimental.v2;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v2";
+
+service WorkspaceService {
+  // GetWorkspace returns a single workspace.
+  //
+  // +return NOT_FOUND User does not have access to a workspace with the given
+  // ID +return NOT_FOUND Workspace does not exist
+  rpc GetWorkspace(GetWorkspaceRequest) returns (GetWorkspaceResponse) {}
+}
+
+message GetWorkspaceRequest { string id = 1; }
+
+message GetWorkspaceResponse { Workspace item = 1; }
+
+// +resource get workspace
+message Workspace {
+  string id = 1;
+  // prebuild indicates it is a prebuild
+  // TODO(ak) model prebuilds as a separate resource
+  bool prebuild = 2;
+
+  string organization_id = 3;
+
+  string name = 4;
+  bool pinned = 5;
+
+  WorkspaceStatus status = 6;
+
+  // additional_environment_variables provide additional environment variables
+  // which take precedence over environment variables provided by the project
+  // and user.
+  //
+  // +optional
+  repeated WorkspaceEnvironmentVariable additional_environment_variables = 7;
```

Comment:
:cloud: All the following fields are part of the "spec", but also mirrored back as part of the "status". We could think about bringing this even closer to the kubernetes-style, and on the top level fully separate the two. But I think that's out of scope here, just leaving it if we think about this again int he future. :+1: